elevate-cpanel

```diff
@@ -1396,7 +1397,8 @@ EOS
             You should disable these repositories and remove packages installed from them
             before continuing the update.
 
-            Consider reporting this limitation to https://github.com/cpanel/elevate/issues
+            Consider reporting this limitation to cPanel Support:
```

Comment:
This should just come out.

Or maybe something like: If you need help removing the packages, support might be able to help... Not sure if this will end with damage and they'd be better off waiting until we support their repo.

Reply:
Removing sounds good given the context. If they can run this on the CLI I would hope they know how to run yum or rpm